modules/test/application/clicommands/PhpCommand.php

```diff
@@ -155,6 +158,96 @@ public function styleAction()
         );
     }
 
+    /**
+     * Run code-validity checks
+     *
+     * This command checks whether icingaweb and installed modules match PHP syntax.
+     *
+     * USAGE
+     *
+     * icingacli test php validity
+     */
+    public function validityAction()
```

Comment:
A CLI Action is like a web Controller, at best it shouldn't contain any business logic. Could you please split this method into multiple ones and move it to a correspondingly named class, maybe in the Icinga\Test namespace?

Reply:
What for? No one else uses that "business logic". (And I hope no one will.)